webroot/src/components/Lists/Pagination/Pagination.spec.ts

```diff
@@ -7,9 +7,9 @@
 
```

Comment:
I have a few questions on these tests, but I think they will mostly be addressed by my comment on `Pagination.ts`.